resources/js/core.js

```diff
@@ -154,6 +155,7 @@ const photoBooth = (function () {
             $('.loading').text(L10N.cheese);
         } else {
             $('.loading').text(L10N.cheeseCollage);
+            $('<p>').text(`${nextCollageNumber + 1} / ${collageLimit}`).appendTo('.loading');
```

Comment:
This adds the picture number to the cheese message and is distinct from what you described in the pr description (at least in my understanding).

Reply:
I updated the description - maybe it makes more sense now?